src/audio/copier/copier.c

```diff
@@ -629,12 +639,13 @@ static int copier_copy(struct comp_dev *dev)
 	return do_multi_endpoint_module_copy(cd, dev);
 }
 
-/* configure the DMA params */
-static int copier_params(struct comp_dev *dev, struct sof_ipc_stream_params *params)
+static int copier_mod_params(struct processing_module *mod)
 {
-	struct processing_module *mod = comp_get_drvdata(dev);
+	struct sof_ipc_stream_params *params = mod->stream_params;
 	struct copier_data *cd = module_get_private_data(mod);
-	int i, ret = 0;
+	struct comp_dev *dev = mod->dev;
+	int ret = 0;
+	int i;
```

Comment:
what was wrong with the original line 637?

Reply:
you are right, nothing changed, let me change back, don't remember why have this change.